lib/utils/config/definitions.js

```diff
@@ -442,7 +441,7 @@ define('ci-name', {
   description: `
     The name of a continuous integration system.  If not set explicitly, npm
     will detect the current CI environment using the
-    [\`@npmcli/ci-detect\`](http://npm.im/@npmcli/ci-detect) module.
+    [\`ci-info\`](http://npm.im/@npmcli/ci-info) module.
```

Comment:
Invalid URL for ci-info. `http://npm.im/ci-info`.

Also here https://github.com/npm/cli/pull/5858/files#diff-6f21b9547f08121788002c1b5494afcd450c3d6a9a6f4e1f452d71d2764515b4R761

Reply:
fiddlesticks.  search and replace failed me, sorry about that.

this file is the source of truth, changing it and running snapshots will fix the rest.